### ISSUE TITLE
'This is likely a bug' when installing click.

### ISSUE THREAD

**Issue by facingBackwards** [kennethreitz/pipenv#859]
I assume this is a known issue since the message suggests it's a bug but installing click showed up the error message (copied below). Running it a second time said it was already installed and completed successfully.

This isn't really an issue for me but I thought I'd dump the stacktrace here on the off chance it's any use to you.

Thanks for pipenv in general. It's an incredibly useful tool.

```
❯ pipenv install --dev click
Installing click…
Collecting click
  Using cached click-6.7-py2.py3-none-any.whl
Installing collected packages: click
Successfully installed click-6.7

Adding click to Pipfile's [dev-packages]…
Locking [dev-packages] dependencies…
Traceback (most recent call last):
  File "/usr/local/bin/pipenv", line 11, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python2.7/dist-packages/pipenv/vendor/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/pipenv/vendor/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/pipenv/vendor/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/dist-packages/pipenv/vendor/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/pipenv/vendor/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/pipenv/cli.py", line 1849, in install
    do_lock(system=system, pre=pre)
  File "/usr/local/lib/python2.7/dist-packages/pipenv/cli.py", line 1024, in do_lock
    pre=pre
  File "/usr/local/lib/python2.7/dist-packages/pipenv/utils.py", line 421, in resolve_deps
    resolved_tree.update(resolver.resolve())
  File "/usr/local/lib/python2.7/dist-packages/pipenv/patched/piptools/resolver.py", line 97, in resolve
    'This is likely a bug.' % max_rounds)
RuntimeError: No stable configuration of concrete packages could be found for the given constraints after 12 rounds of resolving.
This is likely a bug.
```

Running a second time
```
❯ pipenv install --dev click
Installing click…
Requirement already satisfied: click in /home/facingBackwards/.local/share/virtualenvs/project-9BqVMaCB/lib/python3.6/site-packages

Adding click to Pipfile's [dev-packages]…
Locking [dev-packages] dependencies…
Locking [packages] dependencies…
Updated Pipfile.lock (28359e)!
```

```
❯ pipenv --version
pipenv, version 8.2.5
```

**Comment by vphilippon** [comment]
Was there anything else in your Pipfile, other than `click`?
If not, it's really weird, because click has no sub-dependencies, and something is really wrong.
If you had anything else in your `Pipfile` that had dependencies on packages that provides different wheels for different OS or Python version, then its a bug that was fixed in 8.2.7.

In any case, you should upgrade to 8.2.7 and check.

**Comment by facingBackwards** [comment]
I had other stuff in the pip file but it was an existing environment so it was all already installed. I was adding click so wasn't expecting anything else to be installed at the time. 

It's only happened the one time but I'll update and keep an eye out.

**Comment by vphilippon** [comment]
To avoid any confusion, I was referring to the wrong bug in my comment, sorry,
This has nothing to do with the wheel-per-platform bug, but rather another one causing dependency resolution stabilisation issues, which is also fixed in 8.2.7 (#842).
As the locking is redone after each install, any new package could trigger the "random" bug.
I'm pretty sure this should be fixed then 😄

**Comment by erinxocon** [comment]
Should be fixed in 8.2.7!  Closing!
